app/controllers/vendor_api/ping_controller.rb

```diff
@@ -0,0 +1,5 @@
+class VendorApi::PingController < VendorApi::VendorApiBaseController
```

Comment:
This ping is useful for deployments as well — should it be in a more generic namespace?

Reply:
This is just an example to avoid having an empty namespace, we can remove/move it later. We should build that healthcheck endpoint for deployments though: https://trello.com/c/lUM7AMqG/55-add-healthcheck-endpoint-for-the-deploys